app/scripts/first-time-state.js

```diff
@@ -1,7 +1,6 @@
 /**
  * @typedef {object} FirstTimeState
  * @property {object} config Initial configuration parameters
- * @property {object} NetworkController Network controller state
```

Comment:
This type is used in the flow of the database initialization, and `NetworkController` property can never exist on the `FirstTimeState` object; i don't know why it was here.